pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -77,10 +82,12 @@ export const AnalysisWizard: React.FunctionComponent<IAnalysisWizard> = ({
 }: IAnalysisWizard) => {
   const title = "Application analysis";
   const dispatch = useDispatch();
+  const isMutating = useIsMutating();
 
   const [isInitTasks, setInitTasks] = React.useState(false);
   const [createdTasks, setCreatedTasks] = React.useState<Array<Task>>([]);
-  const isMutating = useIsMutating();
+
+  const { hasIdentity, fetchIdentities } = useHasIdentity();
```

Comment:
Could pass in the identities here & create a reusable filter hook that is extensible to other types if you wanted. Otherwise, I think the useHasIdentity may be better off as a one off utility function since it isn't really reused.